rdb/src/main/java/com/instana/dc/rdb/DbDcUtil.java

```diff
@@ -201,10 +209,6 @@ public static String getSimpleStringWithSql(Connection connection, String queryS
     public static <T> List<T> getSimpleListWithSql(Connection connection, String queryStr) {
         try {
             ResultSet rs = executeQuery(connection, queryStr);
-            if (rs.isClosed()) {
```

Comment:
This could happen in our tests. Please do not delete this scenario.

Reply:
Could you please explain the case. 
Because to me if the result set is closed and then it is trying to execute 
`int nColumn = rs.getMetaData().getColumnCount();` 
it will be throwing SQL exception and in the catch block we have already handled that.

Reply:
In our test, this result set can be closed immediately in Oceanbase which is a cloud database. The is to avoid unnecessary long exception messages.

Reply:
done